### DISCUSSION TITLE
Access Exception Title without Stack Trace

### DISCUSSION THREAD

**Discussion by richardlawley** [datalust/seq-tickets#1202] [Help]
I'm looking for a way to get an Exception title to use within an email template.  I'm aware of `{{$Exception}}`, which contains the type and message of an exception followed by the stack trace (I'm only after the first line of this).  I cannot find a way to get the title and message of the Exception alone.

I did also consider using handlebars to take the first line of the exception, but I would need to be able to register a custom helper for that, which I don't believe is possible.

My reason for wanting this is that I would put it as a subject line in an email notification.

**Comment by richardlawley** [comment]
I've realised that this probably isn't possible within Seq, so am implementing it with a Serilog Enricher to add ExceptionMessage and ExceptionType properties.

**Reply by nblumhardt** [reply]
I think that's likely to be the best option :+1: .. It's possible to try various `Substring()` type operations in Seq, but the enricher approach will be much more straightforward and reliable.

https://github.com/RehanSaeed/Serilog.Exceptions is a great pre-built option with various exception logging options.

HTH!
